recipe/meta.yaml

```diff
@@ -29,19 +32,18 @@ build:
 
 requirements:
   build:
-    - curl
-
+    - curl  # [unix]
```

Comment:
Is this still being used? Looks like it might not be.

Reply:
Broke out as issue ( https://github.com/conda-forge/openjdk-feedstock/issues/40 ).